### PR TITLE
20 update readmemd documentation

### LINKED ISSUE #20: Update README.md Documentation
## Requirements

- [ ] Project description
- [ ] Installation process for front and backend
- [ ] Team members
- [ ] Technologies
- [ ] Copyright
- [ ] References/Sources

Example of readme in the past: https://github.com/n2chao/SOEN341

### PR DESCRIPTION
**Pull request by Samuditha04**

### CODE REVIEW

**Review by Samuditha04 — COMMENTED** [review]
Finalized ReadME.md file up for review for all.

**Review by JayC-SF — APPROVED** [review]
Really great job with the Readme, looks very complete to me 💯

File: README.md [hunk 1]
```diff
@@ -1,12 +1,109 @@
...
+
+⚫ Frontend: HTML, CSS, JavaScript, React
+
+⚫ Backend: Java, Go, Node.js
+
+⚫ Database: MongoDB
+
+⚫ Version Control: Git, GitHub
+
+⚫ Other Libraries: Bootstrap
```
Comment (mcndl13) [inline README.md]: Here you should change boostrap to Tailwind CSS.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,109 @@
...
+| Anh Thien Nguyen         | Upon request |
+| Daniel Lam               | 40248073     |
+| Juan-Carlos Sreng-Flores | 40101813     |
+| Ming-Yang Calvin Lee     | 40264581     |
+| Mouhamed Coundoul        | 40248237     |
+| Samuditha Wijenarayana   | 40224895     |
+
+## Technologies
+
+⚫ Frontend: HTML, CSS, JavaScript, React
```
Comment (mcndl13) [inline README.md]: Should we mention Vite here?
Also should it be JavsScript or TypeScript?

**Review by mcndl13 — APPROVED** [review]
Everything seems good, thanks a lot Sam!. Just simply make sure to check the comments I left you and after that, I believe your PR should be done.

File: README.md (hunk 2 above)
Comment (JayC-SF) [reply README.md]: Yeah I guess we can change `React` for `Vite React` and `JavaScript` for `TypeScript`

**Pull request merged**